server.go

```diff
@@ -66,6 +66,29 @@ func main() {
 		// Create the admin web service router
 		handler = CSRF(service.AdminRouter(&env))
 		address = ":8081"
+	case "system-user":
+		// configure request forgery protection
+		csrfSecure := true
+		csrfSecureEnv := os.Getenv("CSRF_SECURE")
+		if csrfSecureEnv == "disable" {
+			log.Println("Disable secure flag")
+			csrfSecure = false
+		}
+
+		CSRF := csrf.Protect(
+			[]byte(env.Config.CSRFAuthKey),
+			csrf.Secure(csrfSecure),
+			csrf.CookieName("XSRF-TOKEN"),
```

Comment:
The default one for Gorilla is starts with and underscore and that seems to cause an issue with retrieving the value. This cookie name is the default for the axios library.